test/framework/framework.go

```diff
@@ -65,22 +65,25 @@ func Init() {
 	initialized = true
 }
 
-// Runs kustomize build followed kubectl apply. dir specifies the name of a test directory.
+// KustomizeAndApply runs kustomize build followed kubectl apply. dir specifies the name of a test directory.
 // By default this function will run kustomize build on dir/overlays/k8ssandra. This will
 // result in using upstream operator images. If you are testing against a fork, then set
 // the TEST_OVERLAY environment variable to specify the fork overlay to use. When
 // TEST_OVERLAY is set this function will run kustomize build on
 // dir/overlays/forks/TEST_OVERLAY which will allow you to use a custom operator image.
-func KustomizeAndApply(namespace, dir string) {
+func KustomizeAndApply(namespace, dir, overlay string) {
```

Comment:
How come you added the `overlay` parameter here? The goal was to have a way for someone to run the tests against his own image. By adding the `overlay` parameter, I will have to modify the test to use a different image. And then I would have to do it for every test.